src/main/cpp/jni/javautilities.cpp

```diff
@@ -226,4 +218,19 @@ bool JavaUtilities::createTableFromFile( jstring tableId, jstring aWAVFilePath )
     return true;
 }
 
+void JavaUtilities::setCpuCores( jintArray cpuCores, jint amountOfCores )
+{
```

Comment:
On Android N we can now store the CPU cores that are exclusive to the foreground application for a consistent performance.